<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgProcesso</class>
 <widget class="QWidget" name="WdgProcesso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>64</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblId">
     <property name="text">
      <string>id</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTempo">
     <property name="text">
      <string>tempo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
